module_7/assignment1_1.js

```diff
@@ -0,0 +1,21 @@
+//  How will you create a new copy of the object below while updating the value of address.details[0] to “5“?
+
+const user = {
+  name: "Harry Potter",
+  age: 12,
+  address: {
+    details: ["4", "Privet Drive"],
+    area: "Little Whinging",
+    city: "Surrey",
+    state: "England",
+  },
+};
+
+//const userClone = JSON.parse(JSON.stringify(user)); // Actual implementation in structuredClone
+const userClone = structuredClone(user); // Nested object - deep clone
```

Comment:
For deep cloning there is other method also JSON.stringify please check that also

Reply:
sure